src/i18n/en.json

```diff
@@ -48,9 +48,9 @@
     "you_want_create_account": "Do not you have an account yet?"
   },
   "reports": {
-    "allow_enable_trial_HTML": "<p>Target users based on their behavior on your website or E-commerce. Thanks to this feature you can send automatic and 100% personalized Emails to the users who visit a specific page. For example, you can send a discount to users who view a particular product page.</p><h4>When does the free trial end?</h4><p>Our goal is that you can see how to use this feature in your business and check its benefits, so for the moment you can use it with no restrictions.</p><h4>What will happen to the Automations created when the free trial ends?</h4><p>They'll remain inactive, and you won't be able to edit them until you hire the special module which includes this service.</p>",
-    "allow_enable_trial_button": "Activate free trial",
-    "allow_enable_trial_title": "Try On-Site Tracking Automation for FREE",
+    "allow_enable_trial_HTML": "<p>Activate the trial period and access Detailed reports on the behavior of users inside your Website or E-commerce. Discover which are the most visited pages, how many visitors have an Email identified by Doppler and how many don't.</p><p> Any doubts? Press <a target=\"_blank\" href=\"https://help.fromdoppler.com/en/advanced-reports\">HELP.</a></p>",
```

Comment:
⚠️ The same issue with the link, please fix before merging.

See https://github.com/FromDoppler/doppler-webapp/pull/166/files#r280847192